examples/defaulter-gen/tests/marker_test.go

```diff
@@ -142,3 +142,16 @@ func Test_Marker(t *testing.T) {
 		})
 	}
 }
+
+func Test_DefaultingFunction(t *testing.T) {
```

Comment:
You could have just added an extra field in the other struct, maybe that's simpler? No strong opinion.

Reply:
Wanted to avoid the defaulting function being executed in the other test since it would apply to every test case and 
 it could be confusing why one of the defaults was changed without any indication in the test name.

Reply:
That makes sense, thanks!